cluster-provision/gocli/cmd/run.go

```diff
@@ -373,8 +373,20 @@ func run(cmd *cobra.Command, args []string) (err error) {
 	wg := sync.WaitGroup{}
 	wg.Add(int(nodes))
 	// start one vm after each other
+
 	for x := 0; x < int(nodes); x++ {
 
+		extraArgs := qemuArgs
```

Comment:
this is still only qemuArgs, until you put it into extra args on 387. Mind keeping it as qemuArgs until put into `"--qemu-args \"" + qmuArgs + "\""`

Reply:
We need a clean variable every iteration so we don't append to next nodes, but you are right this is not extra args, let me rename the var.